README.md

```diff
@@ -13,32 +13,52 @@ To contribute, please check out our contribution guide [here](https://github.com
 
 # Content
 
-- [CSS](#css)
-- [CSharp](#csharp)
-- [Javascript](#javascript)
-- [React](#react)
-- [GitHub](#github)
-- [Magento](#magento)
-- [Fudamental CP algorithms](#fundamental-cp-algorithms)
-  - [Algebra](#algebra)
-  - [Data Structures](#data-structures)
-  - [Dynamic Programming](#dynamic-programming)
-  - [String Processing](#string-processing)
-- [Python](#python)
-- [Android](#android)
-- [C Sharp](#c-sharp)
-- [Others](#others)
-  - [Cheat Sheets](#cheat-sheets)
-  - [Design](#design)
+- [Awesome Resources ![Awesome](https://github.com/shahednasser/awesome-resources)](#awesome-resources-img-srchttpscdnrawgitcomsindresorhusawesomed7305f38d29fed78fa85652e3a63e154dd8e8829mediabadgesvg-altawesome)
```

Comment:
This Link can be removed Not sure what it is @shahednasser

Reply:
I think you added this by mistake. It is not in this repository.